src/scanpy/_settings.py

```diff
@@ -198,31 +200,29 @@ def plot_suffix(self, plot_suffix: str):
         self._plot_suffix = plot_suffix
 
     @property
-    def file_format_data(self) -> str:
+    def file_format_data(self) -> AnnDataFileFormat:
         """File format for saving AnnData objects.
 
-        Allowed are 'txt', 'csv' (comma separated value file) for exporting and 'h5ad'
-        (hdf5) for lossless saving.
+        For lossless saving, use `'h5ad'` (hdf5) or `'zarr'`.
```

Comment:
zarr is usually not losless, and almost certainly not in this case

Reply:
At this point, I also removed the `csv` option, so I can go back with the language.